users/assume_provision_policy.tf

```diff
@@ -0,0 +1,25 @@
+# IAM assume role policy document that allows assumption of the IAM role that
+# allows full IAM administrator permissions
+data "aws_iam_policy_document" "assume_provision_account_role_doc" {
```

Comment:
For consistency, this should be `assume_provisionaccount_role_doc`.  See also line 15.

Reply:
I did the same thing in the "shared-services" directory in #3!  I must remember incorrectly what we agreed to.